client/src/containers/EditTrip/index.js

```diff
@@ -32,8 +36,19 @@ class EditTrip extends PureComponent {
 
     if (updateResult) {
       nextProps.history.push(`${routes.TRIPS}/${trip._id}`);
-    } else {
+    } else if (trip) {
+      let editorState = EditorState.createEmpty();
+      try {
+        const parsedDescription = JSON.parse(trip.description);
+        const contantState = convertFromRaw(parsedDescription);
+        editorState = EditorState.createWithContent(contantState);
+      } catch (e) {
+        console.log('Parsing description error: ', e); // eslint-disable-line no-console
```

Comment:
Later I think it will be needed to show some error or something instead of console.log... 
It is a kind of temporary solution...